src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -50,10 +50,7 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
             builder.UseMiddleware<HttpExceptionMiddleware>();
             builder.UseMiddleware<ResponseBufferingMiddleware>();
             builder.UseMiddleware<HomepageMiddleware>();
-            builder.UseWhen(context => context.Features.Get<IFunctionExecutionFeature>() != null, config =>
-            {
-                config.UseMiddleware<HttpThrottleMiddleware>();
```

Comment:
In the current codebase, this throttle middleware isn't actually active. I believe this is because the way the pipeline adds IFunctionExecutionFeature has changed (added by the ScriptRouteHandler)?